src/app/api/price-search/route.ts

```
@@ -0,0 +1,261 @@
+import { JSDOM } from "jsdom";
+import { translateWhiskyNameToJapenese } from "@/utils/translateWhiskyNameToJapenese";
+import fetch from "node-fetch";
+
+// 캐시 타입 정의
+interface CacheItem {
+  data: any;
+  timestamp: number;
+}
+
+// 캐시를 저장할 객체 (In-Memory)
+let cache: Record<string, CacheItem> = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'cache' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)